### DISCUSSION TITLE
数据存放位置及页面和数据分离的问题

### DISCUSSION THREAD

**Discussion by beita-wang** [alibaba/formily#3824] [Q&A 2.x]
我们使用的是formily2的schema模式开发
1. schema包含页面结构schema和后端数据。想请问后端数据是和页面结构放到一块一起返回，还是数据单独查询后再放到schema结构中。哪种方式更好？
2. 什么样的数据放到dataSource, 什么样的数据放到value中。 或者数据可以都放到value中吗？

**Comment by janryWang** [comment]
1. 一般是两个接口，一个返回页面结构，一个返回页面数据
2. dataSource 主要是选项数据，比如 Select options，value 是表单提交的数据

**Reply by beita-wang** [reply]
如果是类似于tabel的初始化数据该存放到field哪个属性中

**Reply by janryWang** [reply]
放 value
